src/modules/dashboard/Dashboard.utils.ts

```diff
@@ -0,0 +1,60 @@
+import { TrendingChannelsResponse } from 'queries/types';
+import { TrendingChannelsType } from './Dashboard.types';
+
+export const getTrendingChannelsData = (
```

Comment:
<img width="936" alt="Screenshot 2024-06-14 at 4 31 51 PM" src="https://github.com/push-protocol/push-dapp/assets/25508878/2104fda5-f67f-4ae0-9457-b41b22279753">

@mishramonalisha76 the whole file is filled with typescript error, please fix these and use .reduce or .map instead of for loop. Not a good codebase.

Also Please add relevant doc string comments explaining why are we doing this in first place?